webrender/src/frame.rs

```diff
@@ -301,65 +301,86 @@ impl Frame {
             None => return false,
         };
 
-        let layer = self.layers.get_mut(&scroll_layer_id).unwrap();
-        if layer.scrolling.started_bouncing_back && phase == ScrollEventPhase::Move(false) {
-            return false
-        }
+        let scroll_root_id = match scroll_layer_id.info {
+            ScrollLayerInfo::Scrollable(_, scroll_root_id) => scroll_root_id,
+            ScrollLayerInfo::Fixed => unreachable!("Tried to scroll a fixed position layer."),
+        };
 
-        let overscroll_amount = layer.overscroll_amount();
-        let overscrolling = CAN_OVERSCROLL && (overscroll_amount.width != 0.0 ||
-                                               overscroll_amount.height != 0.0);
-        if overscrolling {
-            if overscroll_amount.width != 0.0 {
-                delta.x /= overscroll_amount.width.abs()
+        let mut scrolled_a_layer = false;
+        for (layer_id, layer) in self.layers.iter_mut() {
```

Comment:
Hrm. The issue here is that I want to scroll all layers that have the same scroll root id, instead of just the first one. Otherwise, perhaps I am misunderstanding your suggestion?